### PR TITLE
wip-old-events-cleanup. updated datasource, import events script and …

### PR DESCRIPTION
**Pull request by mkorolyov**
…sample handmade data.

### CODE REVIEW

**Comment by pferrel** [comment]
Should we merge these as you submit the PR or wait for all?

Also, when do you estimate 1-3 will be done?

**Comment by mkorolyov** [comment]
@pferrel 1-3 pushed to PR few days ago.
this pr should be pushed only after 1-3 stage pr will be pushed.

File: engine.json [hunk 1]
```diff
@@ -6,7 +6,12 @@
...
       "name": "sample-handmade-data.txt",
       "appName": "handmade",
-      "eventNames": ["purchase"]
+      "eventNames": ["purchase"],
+      "eventWindow": {
+        "duration": "1 day", // default is to not drop events
+        "removeDuplicates": "true",  // default = false
+        "compressProperties": "true" // default = false
+      }
     }
```
Comment (pferrel) [inline engine.json:15]: ? non-standard json ? does this actually work?

best use `"comment": "some textual comment" or better yet edit the README.md to document. I can do the README.md if you want.

We should create a secondary engine.json like we did for the other integration test. Then copy the user's to a safe location, copy the test engine to engine.json and after the test restore to user-engine.json. See examples/integration-test

File: examples/import_handmade.py [hunk 2]
```diff
@@ -46,6 +51,7 @@ def import_events(client, file):
         entity_id=data[0],
         target_entity_type="item",  # type of item in this action
         target_entity_id=data[2],
+        event_time=timestamp
       )
       print "Event: " + data[1] + " entity_id: " + data[0] + " target_entity_id: " + data[2]
     elif (data[1] == "$set"):  # must be a set event
```
Comment (pferrel) [inline examples/import_handmade.py:57]: This import script is used for the other integration test, does that still work? We may want to have an import per test so they can be altered without worrying about affecting the other tests.

File: src/main/scala/DataSource.scala [hunk 3]
```diff
@@ -78,9 +82,9 @@ class DataSource(val dsp: DataSourceParams)
     }
 
     // aggregating all $set/$unsets for metadata fields, which are attached to items
-    val fieldsRDD = PEventStore.aggregateProperties(
-      appName= dsp.appName,
-      entityType=  "item")(sc)
+    val fieldsRDD = eventsRDD.filter { e =>
+      e.entityType == "item"
+    }.map(e => e.entityType -> PropertyMap(e.properties.fields, DateTime.now(), DateTime.now()))
```
Comment (pferrel) [inline src/main/scala/DataSource.scala:88]: There is no need to aggregate because this was done during cleaning?

**Pull request closed without merging**